brave/browser-laptop
# Notify users when contribution date is pushed back

Resolves #14000

new notification
![image](https://user-images.githubusercontent.com/9574457/39690890-eb120306-51db-11e8-9d2e-120cd11e43e2.png)


## Submitter Checklist:


## Test Plan:
1) add one site to the ledger table (with time spent less then 30min)
2) add funds
3) close browser and change reconcile stamp to yesterday
4) make sure that date is pushed back for 3 days from the stamp that you set and that you see notification as is shown in the print screen (it can be pushed back for 6 days after restart)

## Reviewer Checklist:


Tests
